lib/whatsapp_sdk/resource/error.rb

```diff
@@ -21,10 +21,18 @@ def initialize(field, message)
         end
       end
 
+      class InvalidField < MissingValue; end
```

Comment:
Where is it used?

Reply:
It should be used in `Component` class. I did add this error class but didn't refactor any existing code.